code/app/commodity/server.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/brahma-adshonor/microservice/code/app/commodity/model"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var (
+	db  *gorm.DB
```

Comment:
it seems that these variables is should not efforts the package namespace

Reply:
correct, because first example, try only have functions in one file.
maybe we can try to have more functions next time